migrations/0001_add_tables.up.sql

```diff
@@ -39,4 +39,5 @@ CREATE TABLE IF NOT EXISTS carrier_request (
     CONSTRAINT status
             FOREIGN KEY(status)
             REFERENCES carrier_request_status(id)
-);
\ No newline at end of file
+);
```

Comment:
don't touch those file if you don't need it

Reply:
ok